### PR TITLE
Fixes #19828: RunNuCommandTest fails with timeout on CI

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19828

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/hooks/RunNuCommandTest.scala [hunk 1]
```diff
@@ -66,7 +66,7 @@ class RunNuCommandTest() extends Specification {
       val prog =
         for {
           p <- RunNuCommand.run(Cmd("env", Nil, Map("PATH" -> PATH, "foo" -> "bar")))
-          c <- p.await.timeout(500.millis).notOptional("oups, timed out")
+          c <- p.await.timeout(1500.millis).notOptional("oups, timed out")
```
Comment (ElaadF) [inline webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/hooks/RunNuCommandTest.scala:69]: the failure is on "has only the 3 stdin, stdout, stderr file descriptors" test

Comment (amousset) [reply webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/hooks/RunNuCommandTest.scala:69]: IT gives:

Comment (amousset) [reply webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/hooks/RunNuCommandTest.scala:69]: indeed!

**Comment by amousset** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**